zkevm-node-additions/init_pool_db.sql

```diff
@@ -2,3 +2,7 @@
 -- created via the `POSTGRES_DB` environment variable passed to the docker
 -- container.
 CREATE DATABASE pool_db;
+\connect pool_db;
+CREATE USER pool_user with password 'pool_password';
```

Comment:
Should we be hardcoding the password? At least before we could theoretically have set a secret password via the environment, in production

Reply:
So we don't use the docker at all in production.

Reply:
In production these same creation scripts are bundled into a single file and manually run against a blank RDS instance, then randomized passwords are generated for each account.

Reply:
There is `pool_db` `state_db` and `prover_db` that all have users/passwords that match this pattern. Then in production the users/databases are still named the same but randomized passwords are used.